test_release_common.sh

```diff
@@ -239,6 +251,16 @@ function _test_release_common_is_7_4_u_release {
 	assert_equals "${?}" "${2}"
 }
 
+function _test_release_common_is_dxp_release {
+	LIFERAY_RELEASE_PRODUCT_NAME="${1}"
+
+	echo -e "Running _test_release_common_is_dxp_release for ${LIFERAY_RELEASE_PRODUCT_NAME}.\n"
+
+	is_dxp_release "${LIFERAY_RELEASE_PRODUCT_NAME}"
```

Comment:
@davisaints, do you really need to use `LIFERAY_RELEASE_PRODUCT_NAME` as a parameter here?